config/environment.js

```diff
@@ -16,9 +16,16 @@ module.exports = function(environment) {
     APP: {
       // Here you can pass flags/options to your application instance
       // when it is created
+    },
+
+    'simple-auth': {
```

Comment:
It's being used further down to append properties to the `simple-auth` configuration